tests/test_upgrade_2024_08_06.py

```diff
@@ -0,0 +1,160 @@
+
+
+    # Voting App before
+    voting_proxy = interface.AppProxyUpgradeable(contracts.voting.address)
+    voting_app_from_repo = contracts.voting_app_repo.getLatest()
+    voting_appId = voting_proxy.appId()
+
+    assert voting_app_from_repo[0] == old_voting_app["version"]
+    assert voting_app_from_repo[1] == old_voting_app["address"]
+    assert voting_proxy.implementation() == old_voting_app["address"]
+
+    # TRP Voting Adapter before
+    trp_voting_adapter_address = contracts.trp_escrow_factory.voting_adapter()
+    assert trp_voting_adapter_address == old_trp_voting_adapter_address
+
+    trp_voting_adapter = interface.VotingAdapter(trp_voting_adapter_address)
+    assert trp_voting_adapter.delegation_contract_addr() == ZERO_ADDRESS
+    assert trp_voting_adapter.voting_contract_addr() == VOTING
+
+    # START VOTE
+    if len(vote_ids_from_env) > 0:
+        (vote_id,) = vote_ids_from_env
+    else:
+        tx_params = {"from": LDO_HOLDER_ADDRESS_FOR_TESTS}
+        vote_id, _ = start_vote(tx_params, silent=True)
+
+    vote_tx = helpers.execute_vote(accounts, vote_id, contracts.voting)
+
+    print(f"voteId = {vote_id}, gasUsed = {vote_tx.gas_used}")
+
+    # Voting App after
+    voting_app_from_repo = contracts.voting_app_repo.getLatest()
+
+    assert voting_app_from_repo[0] == updated_voting_app["version"]
+    assert voting_app_from_repo[1] == updated_voting_app["address"]
+    assert voting_proxy.implementation() == updated_voting_app["address"]
+
+    # TRP Voting Adapter after
+    trp_voting_adapter_address = contracts.trp_escrow_factory.voting_adapter()
+    assert trp_voting_adapter_address == updated_trp_voting_adapter_address
+
+    trp_voting_adapter = interface.VotingAdapter(trp_voting_adapter_address)
+    with reverts():
+        trp_voting_adapter.delegation_contract_addr()
+    assert trp_voting_adapter.voting_contract_addr() == VOTING
+
+    # Replacing Rated Labs with MatrixedLink in Lido on Ethereum Oracle set
+
+    # 1) Remove the oracle member named 'Rated Labs' with address
+    #    0xec4bfbaf681eb505b94e4a7849877dc6c600ca3a from HashConsensus for AccountingOracle on Lido on Ethereum
+    assert not accounting_hash_consensus.getIsMember(rated_labs_oracle_member)
+
+    # 2) Remove the oracle member named 'Rated Labs' with address
+    #    0xec4bfbaf681eb505b94e4a7849877dc6c600ca3a from HashConsensus for ValidatorsExitBusOracle on Lido on Ethereum
+    assert not validators_exit_bus_hash_consensus.getIsMember(rated_labs_oracle_member)
+
+    # 5. Add oracle member named 'MatrixedLink' with address
+    #    0x1d0813bf088be3047d827d98524fbf779bc25f00 to HashConsensus for AccountingOracle on Lido on Ethereum Oracle set
+    assert accounting_hash_consensus.getIsMember(matrixed_link_oracle_member)
+
+    # 6. Add oracle member named 'MatrixedLink' with address
+    #    0x1d0813bf088be3047d827d98524fbf779bc25f00 to HashConsensus for ValidatorsExitBusOracle on Lido on Ethereum Oracle set
```

Comment:
Address in this and prev comment is not the same as in matrixed_link_oracle_member var
and list numbering is broken